modules/nw-metallb-installing-operator-cli.adoc

```diff
@@ -79,40 +59,60 @@ NAME               AGE
 metallb-operator   14m
 ----
 
-. Subscribe to the MetalLB Operator.
-
-.. Run the following command to get the {product-title} major and minor version. You use the values to set the `channel` value in the next
-step.
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `metallb-sub.yaml`:
 +
-[source,terminal]
+[source,yaml]
 ----
-$ OC_VERSION=$(oc version -o yaml | grep openshiftVersion | \
-    grep -o '[0-9]*[.][0-9]*' | head -1)
-----
-
-.. To create a subscription custom resource for the Operator, enter the following command:
-+
-[source,terminal]
-----
-$ cat << EOF| oc apply -f -
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
   name: metallb-operator-sub
-  namespace: metallb-system
+  namespace: metallb-system <1>
 spec:
-  channel: "${OC_VERSION}"
+  channel: stable 
   name: metallb-operator
-  source: redhat-operators
+  source: redhat-operators <2>
   sourceNamespace: openshift-marketplace
-EOF
 ----
+<1> Specify the `Namespace` where you want to install the MetalLB Operator. To install the MetalLB Operator in the `openshift-operators` namespace, specify `openshift-operators` in the `Subscription` CR.
+<2> You must specify the `redhat-operators` value.
+
+.. To create the `Subscription` CR, run the following command:
++
+[source,terminal]
+----
+$ oc create -f metallb-sub.yaml
+----
+
+. Optional: To ensure BGP and BFD metrics appear in Prometheus, you can label the namespace as in the following command:
++
+[source,terminal]
+----
+$ oc label ns <namespace> "openshift.io/cluster-monitoring=true"
+----
++
+For example if the MetalLB Operator is installed in the `openshift-operators` namespace run the following command:
```

Comment:
I'd keep suggesting metallb-system as namespace (as we are suggesting here https://github.com/openshift/openshift-docs/pull/45516/files#diff-9a2ee215dd497eaa5b328d65fea3f26f823eb55a11f86e11699e94f24ab6c532R71 )